content/2_Bronze/Intro_Sorting.mdx

```diff
@@ -153,6 +209,53 @@ Two ways to avoid this:
 
 </JavaSection>
 
+</LanguageSection>
+
+### Dynamic Arrays
+
+<LanguageSection>
+
+<CPPSection>
+
+In order to sort a dynamic array, use `sort(v.begin(), v.end())` (or `sort(begin(v),end(v))`). The default sort function sorts the array in ascending order.
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+    vector<int> v{5, 1, 3, 2, 4};
+    sort(v.begin(), v.end());
```

Comment:
should mirror the example above

Reply:
also it bugs me a little that the spacing in these examples is different than the pair example below, but w/e

Reply:
Wait how does it not mirror the example above

Reply:
sorting part of a vector

Reply:
also how is the spacing off?

Reply:
Well, one of the examples has `for (int i : v)` while the one I wrote has `for (pair<int,int> p: v)` (space before colon, doesn't rlly matter). Looks like your java examples use tabs while cpp uses spaces.